modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountEntryLocalServiceImpl.java

```diff
@@ -179,13 +179,29 @@ public AccountEntry addAccountEntry(
 			ServiceContext serviceContext)
 		throws PortalException {
 
+		return addAccountEntry(
+			null, userId, parentAccountEntryId, name, description, domains,
+			emailAddress, logoBytes, taxIdNumber, type, status, serviceContext);
+	}
+
+	@Override
+	public AccountEntry addAccountEntry(
+			String externalReferenceCode, long userId,
+			long parentAccountEntryId, String name, String description,
+			String[] domains, String emailAddress, byte[] logoBytes,
+			String taxIdNumber, String type, int status,
+			ServiceContext serviceContext)
+		throws PortalException {
```

Comment:
@amosfong I think I preferred how you did it before, by performing an add then update with the erc inside the `addOrUpdateAccountEntry` method. This method seems kind of redundant, since it's basically a less useful version of `addOrUpdateAccountEntry`.